www/caddy/src/opnsense/service/templates/Pischem/Caddy/rc.conf.d/caddy

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT THIS FILE -- OPNsense auto-generated file
+{% if helpers.exists('Pischem.caddy.general.enabled') %}
+    {%- set general_enabled = helpers.toList('Pischem.caddy.general.enabled') | first %}
+    {%- if general_enabled == '1' %}
+caddy_enable="YES"
+# Path to the Caddy setup script
+caddy_setup="/usr/local/opnsense/scripts/Pischem/Caddy/setup.sh"
+# Execute the setup script before Caddy starts
+start_precmd="sh ${caddy_setup}"
```

Comment:
you don't need this with "caddy_setup" in place

Reply:
Solved, works as you said without it. Thanks.